routes/tasks.js

```diff
@@ -23,6 +23,20 @@ router.post('/', (req, res, next) => {
   res.json({tasks});
 });
 
+router.put('/:id', (req, res, next) => {
+  const newData = req.body;
```

Comment:
I would call it editedTask instead of newData

Reply:
makes sense, fixed